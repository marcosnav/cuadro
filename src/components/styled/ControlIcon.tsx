import styled from 'styled-components';

const ButtonIcon = styled.img`
  height: 26px;
  width: auto;
`;

ButtonIcon.displayName = 'ButtonIcon';

export default ButtonIcon;
