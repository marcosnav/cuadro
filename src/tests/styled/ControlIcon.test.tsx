import { mount } from 'enzyme';
import 'jest-styled-components';
import React from 'react';
import ControlIcon from './../../components/styled/ControlIcon';

describe('<ControlIcon/>', () => {
  test('renders properly', () => {
    const component = mount(<ControlIcon />);
    expect(component).toHaveStyleRule('height', '26px');
    expect(component).toHaveStyleRule('width', 'auto');
    expect(component).toMatchSnapshot();
  });
});
