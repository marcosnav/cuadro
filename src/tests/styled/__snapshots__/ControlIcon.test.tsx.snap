// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ControlIcon/> renders properly 1`] = `
.c0 {
  height: 26px;
  width: auto;
}

<ButtonIcon>
  <img
    className="c0"
  />
</ButtonIcon>
`;
